### PR TITLE
Layout of detail post

### PR DESCRIPTION
**Pull request by at-minhdao**
## Description
- Layout of detail post
## Reference Requirements
- Product Backlog ID: PBI-10
- Task ID: #10.2

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

##Picture
image

### CODE REVIEW

File: resources/views/backend/posts/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+              </li>
+              <li class="list-group-item">
+                <b>{{ __('posts.like') }}</b>
+                <a class="pull-right">23</a>
+              </li>
+              <li class="list-group-item">
+                <b>{{ __('posts.create_date') }}</b>
+                <a class="pull-right">2018-01-24</a>
+              </li>
+              {{--  <li class="list-group-item">  --}}
```
Comment (at-hieule) [inline resources/views/backend/posts/show.blade.php]: Không dùng thì xóa nhé

File: resources/views/backend/posts/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                <b style="padding-top: 50px">{{ __('posts.content') }}</b>
+              {{--  </li>  --}}
+              {{--  <li class="list-group-item">  --}}
```
Comment (at-hieule) [inline resources/views/backend/posts/show.blade.php]: Xóa đây nữa nhé

**Comment by at-hieule** [comment]
LGTM

File: resources/views/backend/posts/show.blade.php [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+<!-- Content Wrapper. Contains page content -->
+<div class="content-wrapper">
+  <!-- Content Header (Page header) -->
+  <section class="content-header">
+    <h1>
+      {{ __('posts.detail_post') }}
+    </h1>
+    <ol class="breadcrumb">
+      <li>
+        <a href="#">
```
Comment (at-tuanphan) [inline resources/views/backend/posts/show.blade.php]: add route cho các thẻ a luôn đi em

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**